deepchem/models/callbacks.py

```diff
@@ -86,14 +91,16 @@ def __call__(self, model, step):
       for key in scores:
         model._log_scalar_to_tensorboard(key, scores[key],
                                          model.get_global_step())
-    if self.save_dir is not None:
-      score = scores[self.metrics[self.save_metric].name]
-      if not self.save_on_minimum:
-        score = -score
-      if self._best_score is None or score < self._best_score:
-        model.save_checkpoint(model_dir=self.save_dir)
-        self._best_score = score
+    if self.save_dir is not None and (self._best_score is None or
+                                      score < self._best_score):
+      model.save_checkpoint(model_dir=self.save_dir)
     if model.wandb_logger is not None:
       # Log data to Wandb
       data = {'eval/' + k: v for k, v in scores.items()}
       model.wandb_logger.log_data(data, step, dataset_id=id(self.dataset))
+
+  def get_best_score(self):
```

Comment:
Should be `_get_best_score`

Reply:
I think it's intended to be a public method, so the name is fine.  It just needs a docstring.